src/dates.jl

```diff
@@ -0,0 +1,29 @@
+
+
+# Dates.FixedPeriod == Union{Day, Hour, Microsecond, Millisecond, Minute, Nanosecond, Second, Week}
+Quantity(x::Dates.Nanosecond) = x.value * u"ns"
+Quantity(x::Dates.Microsecond) = x.value * u"μs"
+Quantity(x::Dates.Millisecond) = x.value * u"ms"
+Quantity(x::Dates.Second) = x.value * u"s"
+Quantity(x::Dates.Minute) = x.value * u"minute"
+Quantity(x::Dates.Hour) = x.value * u"hr"
+Quantity(x::Dates.Day) = x.value * u"d"
+Quantity(x::Dates.Week) = x.value * u"wk"
+
+uconvert(a::Units, x::Dates.FixedPeriod) = uconvert(a, Quantity(x))
+
+Base.promote_rule(::Type{Quantity{T,D,U}}, ::Type{<:Dates.FixedPeriod}) where {T,D,U} =
+    Quantity{promote_type(T,Int),D,U}
```

Comment:
```suggestion
    Quantity{promote_type(T,Int64),D,U}
```
The types from `Dates` use `Int64` internally, so this promotion rule should probably be changed to reflect that.

Reply:
OK, I hadn't checked.